cmd/evm/internal/t8ntool/execution.go

```diff
@@ -231,7 +233,6 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 
 		txIndex++
 	}
-	statedb.IntermediateRoot(chainConfig.IsEIP158(vmContext.BlockNumber))
```

Comment:
Why did you drop this?

Reply:
Because we will do a commit afterwards, https://github.com/ethereum/go-ethereum/pull/26671/files/a63b92c933e707f1f87b9040da3a32bae90a9061#diff-06c63748460168e12cbf03984f37c000908e0b7ff50ca0dfe28db286a1dfd36eR267 

It's unnecessary to do it, unless for some strong reasons?